### ISSUE TITLE
One more addition would be nice

### ISSUE THREAD

**Issue by tyeeman** [jaredkrinke/md2blog#45]
Thanks for writing this! Very slick. Even though I would like to change the template now and then I understand your reasons. What I would like is to add one more color option - background color. It would be much appreciated. 

I enjoyed reading your dev blog also about md2blog. Well done!

**Comment by jaredkrinke** [comment]
Good point! I should make the background color configurable.

The biggest hurdle is that only "dark" themes currently will work because the stylesheet uses color to distinguish links, headings, etc., and this is usually not sufficient for "light" color schemes. Honestly, the best approach in the long term would probably be for me to switch to some minimal/classless stylesheet that was written by an *actual designer who knows what they're doing* (as opposed to me).

**Comment by tyeeman** [comment]
Yes, have you seen Kev Quirk's website. He wrote simplecss and it just may be what we need here - 

https://simplecss.org/

**Comment by jaredkrinke** [comment]
Thanks! I had also been looking at https://lissomware.github.io/css/ but Simple.css looks better in light mode.

**Comment by tyeeman** [comment]
Here's a site I recently found with nice color and a button to switch between them.

https://darekkay.com/
